brave/README.md

```diff
@@ -166,10 +166,8 @@ Here's an example of a client span:
 // before you send a request, add metadata that describes the operation
 span = tracer.nextSpan().name(service + "/" + method).kind(CLIENT);
 span.tag("myrpc.version", "1.0.0");
-span.remoteEndpoint(Endpoint.newBuilder()
-    .serviceName("backend")
-    .ip("172.3.4.1")
-    .port(8108).build());
+span.remoteServiceName("backend");
+span.parseRemoteIpAndPort("172.3.4.1", 8108);
```

Comment:
The name `parseRemoteIpAndPort` makes me think this function is returning something instead of being a void. What about just `remoteIpAndPort`?

Reply:
the parse part is also used in zipkin2.Endpoint which is trying to be different. It returns true if it was successful. We don't want to trust the string values as one thing that's common is people put hostnames etc where IPs should be. So parse is implying this..

It might be better named validateAndSetRemoteIpAndPort?